packit/api.py

```diff
@@ -73,9 +75,11 @@ def sync_release(self, dist_git_branch: str, version: str = None):
 
             local_pr_branch = f"{full_version}-{dist_git_branch}-update"
             # fetch and reset --hard upstream/$branch?
-            logger.info(f'using "{dist_git_branch}" dist-git branch')
+            logger.info(f"Using {dist_git_branch!r} dist-git branch")
```

Comment:
we should really agree on this: whether we start writing log messages as sentences with capital letters

Reply:
@lachmanfrantisek WDYT?

Reply:
I am using capital letters, but I do not have a strong opinion.

Reply:
okay, so let's do it, since changed all of them :D